src/derivers/bip39.ts

```diff
@@ -114,17 +186,22 @@ export async function deriveChildKey(
   { path, curve, network }: DeriveChildKeyArgs,
   cryptographicFunctions?: CryptographicFunctions,
 ): Promise<SLIP10Node> {
+  assert(
+    path instanceof Uint8Array,
```

Comment:
Is `path` always the seed now? Should we rename?

Reply:
It is. We can't rename it though, since this is based on a standard interface used for BIP-32, BIP-39, CIP-3.

Reply:
Oh I guess we may not want to rename because the other derives implement `deriveChildKey` as a shared interface?

Maybe we should just update the doc string or somehow make it clearer that this has somewhat different behavior

Reply:
Oh wow I should have refreshed😄

Reply:
Can we add a note about it at least?